src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -324,6 +324,19 @@ $(function () {
         var $input = $("input", this);
         if (!$input.prop("checked")) $input.prop('checked', true).trigger("change");
     });
+    $(".accordion-radio input").on("change", function() {
+        // change event on radio-boxes only happens on checked, not uncheck!
+        $(this).closest(".panel").find(":input").each(function() {
+            if (this.hasAttribute("data-required-if-active")) {
+                this.required = this.getAttribute("data-required-if-active");
+            }
+        });
+        // make all other inputs in panels from the same radio-group (input name) optional
+        $(".accordion-radio:has([name=" + this.name + "]:not(:checked)) + div :input").each(function() {
+           this.setAttribute("data-required-if-active", this.required);
```

Comment:
```suggestion
           if (!this.hasAttribute("data-required-if-active")) this.setAttribute("data-required-if-active", this.required);
```
sonst wird es wenn man zwischen zwei gruppen wechselt bei allen übrigen "unbeteiligten" gruppen mit false überschrieben